apps/emqx/src/emqx_connection.erl

```diff
@@ -111,7 +116,10 @@
     limiter_buffer :: queue:queue(pending_req()),
 
     %% limiter timers
-    limiter_timer :: undefined | reference()
+    limiter_timer :: undefined | reference(),
+
+    %% QUIC conn owner pid if in use.
+    quic_conn_pid :: maybe(pid())
```

Comment:
Do we need concern about the Hot-upgrade compatibility once e5.0.0 released?

Reply:
I checked with @id , We don't support hot upgrade for e5.0.x